roles/ceph-update/tasks/main.yml

```diff
@@ -8,69 +8,68 @@
   delay: 60
   delegate_to: "{{ groups['ceph_monitors'][0] }}"
 
-- name: stop ceph mon
-  service: name=ceph-mon-all state=stopped
-  when: "'ceph_monitors' in group_names"
+# tasks of ceph monitors
+- block:
```

Comment:
Reading `site.yml` it appears that this role will only be ran when `ceph.scaleout` or `upgrade_ceph` is set to true. So re-running ursula without either of those arguments would be safe, this entire role will be skipped.